client/src/pages/homepage.js

```diff
@@ -86,24 +86,6 @@ function Homepage() {
     }
   };
 
-  if (loading) {
-    return (
-      <div className="loading-overlay">
-        <img
-          src={"assets/images/logo.png"}
-          alt="loading"
-          className="loading-image"
-        />
-        <div className="loading-text">
-          <span>Loading</span>
-          <span className="loading-dots">.</span>
-          <span className="loading-dots">.</span>
-          <span className="loading-dots">.</span>
-        </div>
-      </div>
-    );
-  }
```

Comment:
was there a reason why this was removed

Reply:
i created a new file and overwrote the old one, i'll add the stuff back

Reply:
i moved it to App.js; now it only runs during the initial load and not on subsequent homepage visits